cmd/multiFaToVcf/testdata/inputMulti.fa

```diff
@@ -0,0 +1,6 @@
+>hg38_chr2
+NNCGGGTCGGCCAGGCACCGGGTCCTCCTTCAGCACTCGCAGCACGCACT
+---GAAGACGCTACGGAACTCTCCGTTTTCCAGACTCG------------
```

Comment:
I see a string of gaps at the end of this sequence, but I don't see a second insertion.  Is that a bug?

Reply:
This program won't call an indel unless if can see the start and end. So alignments starting or ending with gaps in one species shouldn't produce a variant.

Reply:
Yes. I actually left this around in the test file so we could see how variants are called, and make Riley's point above clear to anyone working with this command. And it's based on the start/end of the reference. 
Unless we want to go back and change this functionality so that it is called as a variant?

Reply:
I will log this as an issue.  Ideally I think we would have an option to control this choice.